static/js/compose.js

```diff
@@ -787,7 +787,7 @@ function validate_stream_message() {
     }
 
     var current_stream = stream_data.get_sub(stream_name);
-    var stream_count = current_stream.subscribers.num_items();
+    var stream_count = (current_stream ? current_stream.subscribers.num_items() : 0);
```

Comment:
We probably want to exit this function immediately if current_stream doesn't make sense.